.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.enableCommitSigning": true
```

Comment:
Remove this file please

Reply:
You can also add `.vscode` files to `.gitignore`